### PR TITLE
Views about Complain Page, Change Password Page and Forgot Password Page

### PR DESCRIPTION
**Pull request by amelaabd**

### CODE REVIEW

File: src/OnlineVotingS.API/Models/ChangePasswordViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿namespace OnlineVotingS.API.Models
+{
```
Comment (AdhurimUP) [inline src/OnlineVotingS.API/Models/ChangePasswordViewModel.cs]: remove brackets

File: src/OnlineVotingS.API/Models/ComplainViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿namespace OnlineVotingS.API.Models
+{
```
Comment (AdhurimUP) [inline src/OnlineVotingS.API/Models/ComplainViewModel.cs]: remove brackets

File: src/OnlineVotingS.API/Models/ForgotPasswordViewModel.cs [hunk 3]
```diff
@@ -0,0 +1,7 @@
+﻿namespace OnlineVotingS.API.Models
+{
```
Comment (AdhurimUP) [inline src/OnlineVotingS.API/Models/ForgotPasswordViewModel.cs]: remove brackets

**Review by AdhurimUP — CHANGES_REQUESTED** [review]
resolve comments

File: src/OnlineVotingS.API/Models/ChangePasswordViewModel.cs (hunk 1 above)
Comment (amelaabd) [reply src/OnlineVotingS.API/Models/ChangePasswordViewModel.cs]: done

**Review by AdhurimUP — APPROVED** [review]

**Pull request merged**